include/rdma/fi_domain.h

```diff
@@ -175,6 +175,7 @@ struct fi_mr_attr {
 		int		synapseai;
 	} device;
 	void			*hmem_data;
+	size_t			page_size_hint;
```

Comment:
This should be page_size

Reply:
This breaks the ABI and requires some sort of versioning checks in all the providers and relevant updates to the man page.

Reply:
Yep. I'll add in the ABI compat stuff once we agree on API.